### PR TITLE
Remove obsolete code

### PR DESCRIPTION
**Pull request by xuebingli**
### Short Description

I noticed some obsolete code while I worked on #1250. The commit messages have more information. It's recommended to review commit by commit.

### Pull Request Checklist: 

- [x] Please make sure all jobs pass before requesting a review. 
- [x] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
- [x] Have you lint your code locally before submission?
- [x] Did you write tests where appropriate?

### Related Issue?

N/A

### Screenshots (if appropriate):

N/A

### CODE REVIEW

**Comment by xuebingli** [comment]
Weird that `Common Server Test` failed. I did run the tests successfully before submitting this PR. I'll investigate further tomorrow.

**Comment by xuebingli** [comment]
I tried and could not reproduce the failed test in local dev environment. Investigating further.

File: CommonServer/Services/DatabaseService.ts [hunk 1]
```diff
@@ -1298,13 +1298,6 @@ class DatabaseService<TBaseModel extends BaseModel> extends BaseService {
                 updateBy.limit = new PositiveNumber(updateBy.limit);
             }
 
-            const select: Select<TBaseModel> = {};
```
Comment (simlarsen) [inline CommonServer/Services/DatabaseService.ts:1301]: Why remove this?

Comment (xuebingli) [reply CommonServer/Services/DatabaseService.ts:1301]: While `select` is assigned and populated, it doesn't appear to be used/referenced anywhere, so I think it's safe to remove. 

Please correct me if I'm wrong. I'm still new to the code base.

**Pull request merged**